types/mui-datatables/index.d.ts

```diff
@@ -164,8 +177,21 @@ export interface MUIDataTableFilterOptions {
 }
 
 export interface MUIDataTableCustomFilterListOptions {
+    /**
+     * Function that return a string or array of strings use as the chip label(s)
+     * @link https://github.com/gregnb/mui-datatables/blob/master/examples/customize-filter/index.js
```

Comment:
```suggestion
     * {@link https://github.com/gregnb/mui-datatables/blob/master/examples/customize-filter/index.js Example}
```

This way the clickable link in IntelliSense will just say "Example" and actually function as TSDoc link